### PR TITLE
Nzcsa 106 member discount page

### PR DESCRIPTION
**Pull request by KirstyGong**

### CODE REVIEW

**Review by nancy111573 — APPROVED** [review]

File: src/pages/Dashboard/SponsorGrid.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    padding: theme.spacing(2),
+  },
+}));
+
+export default function SponsorsLogoLayout() {
+
+  let s = '';
```
Comment (Alexl-268) [inline src/pages/Dashboard/SponsorGrid.js:17]: Could we have a name for s

File: src/assets/storeData.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
+const data = [
```
Comment (Alexl-268) [inline src/assets/storeData.js:1]: maybe we can add this to the backend in the future

**Review by Alexl-268 — CHANGES_REQUESTED** [review]
Please have a name for variable s in src/pages/Dashboard/SponsorGrid.js

File: src/pages/Dashboard/SponsorGrid.js (hunk 1 above)
Comment (KirstyGong) [reply src/pages/Dashboard/SponsorGrid.js]: idk just let it be for now

File: src/assets/storeData.js (hunk 2 above)
Comment (KirstyGong) [reply src/assets/storeData.js:1]: yup

**Review by Alexl-268 — APPROVED** [review]

**Pull request merged**